apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -127,6 +127,13 @@ const getBlockInserterUsed = ( originalBlockIds = [] ) => {
 		return 'quick-inserter';
 	}
 
+	// This checks validates if we are inserting a block from the Payments Inserter block.
+	if (
+		select( 'core/block-editor' ).getBlockName( originalBlockIds[ 0 ] ) === 'jetpack/payments-intro'
+	) {
+		return 'payments_intro_block';
```

Comment:
Let's use kebab-case to keep it consistent with the other return values:
`payments-intro-block` (here and in the docblock)

Reply:
Sure! Done.